sonic-stormond/scripts/stormond

```diff
@@ -338,6 +329,10 @@ class DaemonStorage(daemon_base.DaemonBase):
     # Main daemon logic
     def run(self):
 
+        # Connect to CONFIG_DB and get polling and sync intervals --
+        # this is to be able to dynamically configure the polling and sync times.
+        self._get_configdb_intervals()
```

Comment:
Just to clarify, we need to update the design doc stating that any changes to intervals will take place from next cycle and the if the user wants immediate effect they should perform config reload or some other restart mechanism

Reply:
Understood, will update the HLD right now.

Reply:
@dgsudharsan pls help review https://github.com/sonic-net/SONiC/pull/1703 that describes this redesign.

Reply:
This PR is merged. Resolving.